src/app/shared/filesystem/file-browser/file-browser.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="top-wrapper">
-  <clark-file-breadcrumb [paths$]="currentPath$"></clark-file-breadcrumb>
+  <clark-file-breadcrumb [paths]="currentPath" (pathChanged)="handlePathChanged($event)"></clark-file-breadcrumb>
   <div class="view-switcher inline-flex right">
     <div class="inline-flex">
-      <button *ngIf="canManage && ((currentNode$| async)?.children.length || (currentNode$| async)?.files.length)" id="new-file" (click)="emitNewOptionClick($event)" class="button new">
+      <button *ngIf="canManage && ((currentNode$| async)?.folders.length || (currentNode$| async)?.files.length)" id="new-file" (click)="emitNewOptionClick($event)" class="button new">
         New Upload <i class="fas fa-upload"></i>
       </button>
       <!-- <span *ngIf="view==='list'" (click)="view='grid'" class="button clean">
```

Comment:
Does this component still have the grid view, we just don't allow it to be turned on?

Reply:
Good catch! We don't have the grid view anymore. I'll remove the commented code here.